source/common/common/logger.h

```diff
@@ -123,7 +124,66 @@ template <Id id> class Loggable {
 
 } // Logger
 
-#ifdef NDEBUG
+/**
+ * Base logging macros.  It is expected that users will use the convenience macros below rather than
```

Comment:
P.S. I don't care one way or the other, but you had mentioned this previously. can also do in follow up later.